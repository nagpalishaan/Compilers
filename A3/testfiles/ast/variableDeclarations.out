Program
  List
    Function
      IdDecl ID="a"
      Opt
      Opt
        List
          DeclareVar
            IdDecl ID="x"
            Opt
          DeclareVar
            IdDecl ID="y"
            Opt
