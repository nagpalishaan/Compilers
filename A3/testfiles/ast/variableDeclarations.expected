Program
  List
    Function
      IdDecl ID="a"
      ParamList
        List
      StmtBlock
        List
          DeclareVar
            IdDecl ID="x"
            Opt
          DeclareVar
            IdDecl ID="y"
            Opt
